frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -255,8 +255,8 @@ const ActivitySummary = ({
         {
         showGroupInfo && (
         <USWDSAlert type="info">
```

Comment:
This alert is a good candidate for a separate component - <GroupAlert resetGroup={resetGroup} />

That way future text changes don't have to be duped out

Reply:
will do.

Reply:
done